### DISCUSSION TITLE
Re: Multi-Label classification

### DISCUSSION THREAD

**Discussion by chewyh26** [mrdbourke/pytorch-deep-learning#317] [Q&A]
In the starting video of the PyTorch neural network classification, the 3 different problem types binary, multi-class and multi-label classification were mentioned but after reviewing the entire section, I did not see any instructional videos covering the third problem type: multi-label classification. Will you be able to show an example dataset of how pyTorch can be implemented to address the multi-label classification problem type?

**Comment by karthikbhandary2** [comment]
Hello @chewyh26 

I too just went through that part and the multi-label classification problem was not demonstrated. I think it might be covered in the future section since I too am in the middle of learning. If you are interested then you can also do your own research and implement it.
